### PR TITLE
Add some helpers in CountryProvider

### PR DESCRIPTION
**Pull request by MrGrabazu**
I'm using this awesome library and I think those helpers may be a cool add :
- get country list
- get translated name from country code and language code (if exists)
- get translated name from country code and CultureInfo (if exists ro default)

### CODE REVIEW

File: src/Nager.Country/CountryProvider.cs [hunk 1]
```diff
@@ -310,5 +317,99 @@ public ICountryInfo GetCountry(Alpha3Code alpha3Code)
...
+        }
+
+        public string GetCountryTranslatedName(Alpha3Code alpha3Code, CultureInfo culture)
+        {
+            return this.GetCountryTranslatedName(alpha3Code, culture.TwoLetterISOLanguageName);
+        }
+
+        public string GetCountryTranslatedName(string alpha2or3Code, CultureInfo culture, LanguageCode defaultLanguageCode)
+        {
+            string name = this.GetCountryTranslatedName(alpha2or3Code, culture);
```
Comment (tinohager) [inline src/Nager.Country/CountryProvider.cs]: use `var` instead of string

File: src/Nager.Country/CountryProvider.cs [hunk 2]
```diff
@@ -310,5 +317,99 @@ public ICountryInfo GetCountry(Alpha3Code alpha3Code)
...
+            if (String.IsNullOrWhiteSpace(name) == true)
```
Comment (tinohager) [inline src/Nager.Country/CountryProvider.cs]: use lowercase `string.IsNullOrWhiteSpace` don`t use == true

if (string.IsNullOrWhiteSpace(name))

File: src/Nager.Country/CountryProvider.cs [hunk 3]
```diff
@@ -310,5 +317,99 @@ public ICountryInfo GetCountry(Alpha3Code alpha3Code)
...
+            string name = this.GetCountryTranslatedName(alpha2Code, culture);
+            if (String.IsNullOrWhiteSpace(name) == true)
+                name = this.GetCountryTranslatedName(alpha2Code, defaultLanguageCode);
+            return name;
+        }
+
+        public string GetCountryTranslatedName(Alpha3Code alpha3Code, CultureInfo culture, LanguageCode defaultLanguageCode)
+        {
+            string name = this.GetCountryTranslatedName(alpha3Code, culture);
+            if (String.IsNullOrWhiteSpace(name) == true)
```
Comment (tinohager) [inline src/Nager.Country/CountryProvider.cs]: please use on all `if` curved bracket

File: src/Nager.Country/CountryProvider.cs [hunk 4]
```diff
@@ -310,5 +317,99 @@ public ICountryInfo GetCountry(Alpha3Code alpha3Code)
...
+        }
+
+        public string GetCountryTranslatedName(Alpha2Code alpha2Code, LanguageCode languageCode)
+        {
+            return GetCountryTranslatedName(GetCountry(alpha2Code), languageCode);
+        }
+
+        public string GetCountryTranslatedName(Alpha3Code alpha3Code, LanguageCode languageCode)
+        {
+            return GetCountryTranslatedName(GetCountry(alpha3Code), languageCode);
```
Comment (tinohager) [inline src/Nager.Country/CountryProvider.cs]: use `this`

**Comment by tinohager** [comment]
@MrGrabazu the commit have failures check AppVeyor log

**Comment by MrGrabazu** [comment]
I don't touch this test part, the data validation fail because the source
of data used to compare has changed.

**Pull request merged**